docs/reference/model.rst

```diff
@@ -1,7 +1,82 @@
-.. automodule:: stravalib.model
-   :members:
-   :undoc-members:
-   :show-inheritance:
+====================================
+Model Module: Functions and Classes
+====================================
+.. currentmodule:: stravalib.model
 
+The ``Model`` does special things
```

Comment:
Can we generate this from module-level docstrings* so we can have a single source of truth?

(*) I know these don't exist everywhere yet, but maybe it's a good idea to add them?

Reply:
we can! that is why i just added stub / random language. I think the more that we add to docstrings, module description, etc the better all around. so i'll do that on the next round of commits to this pr:

What i'll do is first look at errors and missing things. then i'll go through and update module descriptions .

maybe then we merge and i can work on a module at a time to update docstrings